specification/vmwarecloudsimple/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+```yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "vmwarecloudsimple"
+  output-folder: "$(typescript-sdks-folder)/packages/vmwarecloudsimple"
```

Comment:
Update the output folder to follow the new folder structure in azure-sdk-for-js:
```suggestion
  output-folder: "$(typescript-sdks-folder)/sdk/vmwarecloudsimple/arm-vmwarecloudsimple"
```